Lifecycle.md

```diff
@@ -0,0 +1,279 @@
+# Planetary Software Organization Project and Working Group Lifecycle
+
+## Definitions
+
+The Planetary Software Organization hosts several Top-Level Projects
+(TLPs). These projects are autonomous from each other and governed
+by their own TC (Technical Committee) and chartered by the Planetary
+Software TSC.  Typically, TLPs are software projects (but not always)
+and are typically long-lived and open-ended.
+
+The Planetary Software Organization also hosts several Top-Level Working
+Groups (TLWGs).  TLWGs are typically formed for a specific purpose,
+and when that purpose is achieved, the TLWG will stand down.
+
+TLPs are free to create their own Working Groups which are autonomous groups
+collaborating to fulfill a set of responsibilities. Working Groups are
+eventually chartered by their project TC.
+
+```
+ TSC
+  |
+  |-- Project A TC (Chartered By TSC)
+  |       |-- Working Group (Chartered By Project A TC)
+  |
+  |-- Project B TC (Chartered By TSC)
+  |       |-- Working Group (Chartered By Project B TC)
+  |
+  |-- Working Group X (Chartered by TSC)
+  |-- Working Group Y (Chartered by TSC)
+```
+
+Both TLPs and TLWGs may elect a representative to the TSC. TLPs and WGs
+with *incubation* status are not granted voting privileges on the TSC.
+
+## Lifecycle Overview
+
+In general, a TLP or TLWG (hereafter TL*) will have the following lifecycle:
+
+	1. Like-minded individuals form a TC in order to establish a TL*
+	2. That TC establishes a charter for their proto-TL*
+	3. That TC submits an application to the Planetary Software TSC
+	4. If the application is accepted by the TSC, the proto-TL* enters Incubation
+	5. The TSC works with the TC to get the proto-TL* graduated from Incubation
+	6. The proto-TL* is now a fully authorized Planetary Software TL*
```

Comment:
`proto-TL*` is ambiguous here. I would just say `TL*`. It is already clear that the project or working group is not fully authorized by the PS TSC.

Reply:
Okay.